<launch>
<node pkg="imu_ekf_ros" type="ahrs_initialization.py" name="ahrs_initialization_server"  output="screen" />
<node pkg="tf" type="static_transform_publisher" name="world_to_ENU_tf" args="0 0 0 0 0 0 /map /ENU 100"/>
<node pkg="encoder_ekf_ros" type="aided_nav.py" name="aided_nav"  output="screen" />
<!-- 
<node type="rviz" name="rviz" pkg="rviz" args="-d encoder_ekf_ros/rviz/aided_nav.rviz" /> -->
</launch>
